src/amo/pages/Block/index.js

```diff
@@ -102,6 +102,23 @@ export class BlockBase extends React.Component<InternalProps> {
     return content;
   }
 
+  renderBlockData() {
```

Comment:
We can call this method: `renderVersions`

Reply:
Ohk!